framework-packages.json

```diff
@@ -1,33 +1,45 @@
 {
     "all": {
+        "include": [
+                "fprintd"
+        ],
+        "exclude": []    
+    },
+    "intel": {
         "include": {
-            "all": [
-                "fprintd",
+            "packages": [
                 "tlp",
                 "tlp-rdw",
                 "powertop"
-            ]
+            ],
+            "38-packages": [],
+            "39-packages": []            
         },
         "exclude": {
-            "all": [
+            "packages": [
                 "power-profiles-daemon"
-            ]
-        }
-    },
-    "38": {
-        "include": {
-            "all": []
-        },
-        "exclude": {
-            "all": []
+            ],
+            "38-packages": [],
+            "39-packages": []            
         }
     },
-    "39": {
+    "amd": {
+	"note-ref": "https://knowledgebase.frame.work/en_us/optimizing-ubuntu-battery-life-Sye_48Lg3",    
         "include": {
-            "all": []
+            "packages": [
+                "power-profiles-daemon"
+            ],
+            "38-packages": [],
+            "39-packages": []            
         },
         "exclude": {
-            "all": []
-        }
+            "packages": [
```

Comment:
I don't believe these packages would need to be explicitly excluded as they aren't in the main image